runtime/src/main/scala/akka/grpc/javadsl/GrpcMarshalling.scala

```diff
@@ -14,7 +14,7 @@ import akka.grpc._
 import akka.grpc.internal._
 import akka.grpc.GrpcProtocol.{ GrpcProtocolReader, GrpcProtocolWriter }
 import akka.http.javadsl.model.{ HttpRequest, HttpResponse }
-import akka.japi.Function
+import akka.japi.function.Function
```

Comment:
Here we promised binary compatibility, so we cannot 'just' change from `akka.japi.Function` to `akka.japi.function.Function`.

To try and not bite off too much at once, we could keep `akka.japi.Function` for the eHandler (also in the code generators). Later we plan to replace that with a dedicated class anyway (see #928)

Reply:
Well, I can have a look on that issue first.